projects/js-toolkit/packages/js-toolkit-core/src/index.ts

```diff
@@ -87,10 +87,18 @@ export {default as wrapModule} from './transform/js/operation/wrapModule';
 // JSON source transformation
 
 export * from './transform/json';
+export {default as addConfigurationField} from './transform/json/operation/addConfigurationField';
 export {default as addPkgJsonDependencies} from './transform/json/operation/addPkgJsonDependencies';
+export {default as addPkgJsonScripts} from './transform/json/operation/addPkgJsonScripts';
```

Comment:
With #564 being worked on, shouldn't we avoid abbreviations `pkg` => `package`?

Reply:
This is interesting, because bundler 2 and 3 where full of abbreviations like:

- `pkg` instead of `package` (which is a reserved word)
- `pkgJson` instead of `packageJson` because we use `pkg` instead of `package` and because it's easier to type and understandable (IMO)
- The implicit naming convention for relative/absolute paths ([this](https://github.com/liferay/liferay-frontend-projects/pull/562#discussion_r646538403)) which could lead to very long variables like `indexJsTemplateProjectRelativePosixPath`
- I'm sure there are more... :sweat_smile: 

So, we may change the codebase, but it will require a full review of the code and probably breaking the current APIs (for bundler 3 this is not a big problem, but we shouldn't do it for bundler 2).

Reply:
I'm leaving this as unchanged for now. Once #564 is in place we may decide what to do with bundler v3 (before releasing a following version).